util/kustomize/kustomize.go

```diff
@@ -43,15 +59,15 @@ func (k *kustomize) Build(opts KustomizeBuildOpts, overrides []*v1alpha1.Compone
 	}
 
 	for _, override := range overrides {
-		cmd := exec.Command("kustomize", "edit", "set", "imagetag", fmt.Sprintf("%s:%s", override.Name, override.Value))
+		cmd := exec.Command(commandName, "edit", "set", "imagetag", fmt.Sprintf("%s:%s", override.Name, override.Value))
```

Comment:
kustomize 2 no longer has `edit set imagetag` and has:
```
$ kustomize-2.0.1 edit set --help
Sets the value of different fields in kustomization file.

Usage:
  kustomize edit set [command]

Examples:

	# Sets the nameprefix field
	kustomize edit set nameprefix <prefix-value>

	# Sets the namesuffix field
	kustomize edit set namesuffix <suffix-value>


Available Commands:
  image       Sets images and their new names, new tags or digests in the kustomization file
  nameprefix  Sets the value of the namePrefix field in the kustomization file.
  namespace   Sets the value of the namespace field in the kustomization file
  namesuffix  Sets the value of the nameSuffix field in the kustomization file.
```

Reply:
I'm actually not sure what we want to do here. Do we want to return an error (i.e. leave it alone?). Do we want to ignore it and log a warning?

Reply:
If we know that this is a kustomize 2 app, we skip the `edit set` command entirely. We will handle this later when we introduce kustomize specific config.